meetings/fedora.py

```diff
@@ -1,5 +1,17 @@
+from .util import get_room_alias
+
 async def startmeeting(meetbot, event):
-  meetbot.log.info(f'Fedora: Meeting started in {event.room_id}')
+  room_alias = await get_room_alias(meetbot.client, event.room_id)
+  meetbot.log.info(f'Fedora: Meeting started in {room_alias}')
 
 async def endmeeting(meetbot, event, meeting_id):
-  meetbot.log.info(f'Fedora: Meeting ended in {event.room_id}')
+  room_alias = await get_room_alias(meetbot.client, event.room_id)
+  
+  full_log    = await meetbot.get_items(meeting_id)
+  info_list   = await meetbot.get_items(meeting_id, "info")
+  action_list = await meetbot.get_items(meeting_id, "action")
+
+  await meetbot.upload_file(event, info_list, "info_items.txt")
```

Comment:
Upload_file still defined in the main meetbot module, isn't it? Should probably move here, since I don't think it's used by the discourse uploader.